.drone.yml

```diff
@@ -54,6 +54,17 @@ pipeline:
       # sbt scripted tests are slow and don't run on PRs
       event: [ tag, deployment ]
 
+  test_scripts:
+    group: test
+    image: lampepfl/dotty:2018-01-17
+    commands:
+      - cp -R . /tmp/5/ && cd /tmp/5/
+      - ./project/scripts/sbt ";dist-bootstrapped/pack"
+      - ./dist-bootstrapped/target/pack/bin/dotc tests/run/hello.scala
+      - ./dist-bootstrapped/target/pack/bin/dotr Test
+      - echo ":quit" | ./dist-bootstrapped/target/pack/bin/dotr
+      - ./dist-bootstrapped/target/pack/bin/dotd -project Hello tests/run/hello.scala
+
```

Comment:
@allanrenucci I'm not sure if it's the best way to do it, could you please advise?

Reply:
I would probably merge this with `project/scripts/sbt/sbtTests`. The script tests the commands but within sbt.